src/opentelemetry/launcher/configuration.py

```diff
@@ -289,7 +289,13 @@ def _validate_service_name(service_name: Optional[str]):
 
 class LightstepLauncherInstrumentor(BaseInstrumentor):
     def _instrument(self, **kwargs):
-        configure_opentelemetry()
+        try:
+            configure_opentelemetry()
+        except InvalidConfigurationError as err:
+            _logger.error(
```

Comment:
Would it be better to use [`exception`](https://docs.python.org/3/library/logging.html#logging.Logger.exception) instead?